### DISCUSSION TITLE
Does every plan require all three files?

### DISCUSSION THREAD

**Discussion by todd-kooser** [CMSgov/price-transparency-guide#43] [Q&A] [answered]
We have groups that use our standard PPO network and pricing for medical claims, but may use a different pharmacy manager or process out-of-network claims differently. Are we required to copy and paste our in-network file for each of these groups, or can we provide just the files that are different and point people to the standard PPO in-network file?

**Comment by shaselton-usds** [comment, marked as answer]
I'm not quite understanding the scenario here. What do you mean by "groups"?

Each plan is required to report and not the "groups" that use those plans (I think this is what you mean by "group").

**Reply by todd-kooser** [reply]
By groups I mean employer groups for our commercial business.

Our niche in the commercial market is having highly customizable plan options for employer groups. So, for example, if an employer group comes to us and wants to use our PPO network but a different PBM, I'm assuming we'd need to represent that as a distinct plan that is only applicable to one group because it causes a difference in pricing on the pharmacy side. Since that distinct plan has the exact same in-network and allowed-amount data as our PPO product, can we somehow reference that product and indicate that there is a difference only in Pharmacy information, or do we need to copy and paste the in-network and allowed-amount files so that the plan has all three files represented?

Similarly, some groups will come to us wanting to use the PPO network but with specific pricing negotiated with one provider. In that case I'm assuming we'll need to create a distinct plan for that group, duplicate the PPO plan file and change the cost values for the one provider, but do we need to also copy and paste the allowed-amount and pharmacy files or for that plan can we reference the allowed-amount and pharmacy files for the PPO product?

Does that make sense?

**Reply by shaselton-usds** [reply]
Sorry for the delay! I think some recent updates actually address this concern. Take a look at the changes found here: https://github.com/CMSgov/price-transparency-guide/commit/1e33f28c4c5581790ca031223496f64f990efd11#diff-5788d606f767177de0b701c945124d35072c320957a18a6d7a3db2e52f11e442 It allows for multiple plans to be reported in a single file.
